packages/styling/src/styles/getFocusStyle.ts

```diff
@@ -36,3 +36,28 @@ export function getFocusStyle(
     })
   );
 }
+
+export function getFocusStyleBasedOnAncestorClass(
```

Comment:
please ping @dzearing about focus styles, I'm not quite clear on how we do it in Fabric.